### PR TITLE
MTM-60386-Missing-info-about-required-permissions-to-use-Smart-Rules-relying-on-SMS-provider

### PR DESCRIPTION
**Pull request by pmich-sag**
Screenshot from 2024-09-24 14-18-24
Screenshot from 2024-09-24 14-18-49

### CODE REVIEW

File: content/cockpit/smart-rules-collection.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ In certain rule parameters, various trigger fields can be used as variables, see
 If an alarm is created, an SMS is sent.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+ This rule is available when your tenant has: 
+- configured SMS provider
+- READ permission for the permission type "Option management"
```
Comment (pawel-rynarzewski-c8y) [inline content/cockpit/smart-rules-collection.md]: I think that SMS provider requirement applies to the tenant, but READ permission applies to the user, so maybe better:

```suggestion
This smart rule is only available if:
- your tenant has configured an SMS provider
- your user has a READ permission for the permission type "Option management"
```
?

File: content/cockpit/smart-rules-collection.md [hunk 2]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
 If an alarm is created, sends email or SMS.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+ This rule is available when your tenant has: 
+- configured SMS provider
+- READ permission for the permission type "Option management"
```
Comment (pawel-rynarzewski-c8y) [inline content/cockpit/smart-rules-collection.md]: See above.

File: content/cockpit/smart-rules-collection.md [hunk 3]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
 If an alarm is created, sends email or SMS.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+ This rule is available when your tenant has:
```
Comment (pmich-sag) [inline content/cockpit/smart-rules-collection.md:160]:
```suggestion
This smart rule is only available if:
- your tenant has configured an SMS provider
- your user has a READ permission for the permission type "Option management"
```

File: content/cockpit/smart-rules-collection.md (hunk 2 above)
Comment (pmich-sag) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
This smart rule is only available if:
- your tenant has configured an SMS provider
- your user has a READ permission for the permission type "Option management"
```

Comment (pmich-sag) [reply content/cockpit/smart-rules-collection.md]: Ok, will do the same.

File: content/cockpit/smart-rules-collection.md [hunk 4]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
 If an alarm is created, sends email or SMS.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+This smart rule is only available if:
+- your tenant has configured an SMS provider
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your tenant has configured an SMS provider.
```

File: content/cockpit/smart-rules-collection.md [hunk 5]
```diff
@@ -155,7 +157,9 @@ For details on enabling/disabling a smart rule, see <a href="/cockpit/smart-rule
...
+- your user has a READ permission for the permission type "Option management"
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your user has a READ permission for the permission type "Option management".
```

File: content/cockpit/smart-rules-collection.md [hunk 6]
```diff
@@ -25,7 +25,9 @@ In certain rule parameters, various trigger fields can be used as variables, see
 If an alarm is created, an SMS is sent.
 
 {{< c8y-admon-req >}}
-This rule is only available if your tenant has a configured SMS provider.
+This smart rule is only available if:
+- your tenant has configured an SMS provider
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your tenant has configured an SMS provider.
```

File: content/cockpit/smart-rules-collection.md [hunk 7]
```diff
@@ -25,7 +25,9 @@ In certain rule parameters, various trigger fields can be used as variables, see
...
+- your user has a READ permission for the permission type "Option management"
```
Comment (MWindel) [inline content/cockpit/smart-rules-collection.md]:
```suggestion
- Your user has a READ permission for the permission type "Option management".
```

**Review by pawel-rynarzewski-c8y — APPROVED** [review]
Approved, please check @MWindel comments.

**Comment by pmich-sag** [comment]
I closed this pull request, because there is duplicated pull request there: https://github.com/SoftwareAG/c8y-docs/pull/2457/files created by my second account.

**Comment by pmich-sag** [comment]
> I closed this pull request, because there is duplicated pull request there: https://github.com/SoftwareAG/c8y-docs/pull/2457/files created by my second account.



> I closed this pull request, because there is duplicated pull request there: https://github.com/SoftwareAG/c8y-docs/pull/2457/files created by my second account.

I closed wrong pull request. 
I'm closing this pull request https://github.com/SoftwareAG/c8y-docs/pull/2457 because my new github user does not has permission to c8y-docs repository. 


To sum up: this pull request is valid.

**Review by darlucumulo — APPROVED** [review]

**Review by pawel-rynarzewski-c8y — APPROVED** [review]

**Review by darluk — APPROVED** [review]

**Review by pawel-rynarzewski-c8y — APPROVED** [review]

**Review by darluk — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**